core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -671,7 +669,8 @@ private StorageDirView allocateSpace(long sessionId, AllocateOptions options)
           // Skip the review as we want the allocation to be in the place we just freed
           dirView = mAllocator.allocateBlockWithView(sessionId, options.getSize(),
               BlockStoreLocation.anyTier(), allocatorView.refreshView(), true);
-          LOG.debug("Allocation after freeing space for block creation: {}", dirView);
+          LOG.debug("Allocation after freeing space for block creation. availableBytes: {}",
+                  dirView == null ? 0 : dirView.getAvailableBytes());
```

Comment:
nit: reduce the indent by 2 spaces to be aligned to previous lines

Reply:
done